grails-app/controllers/org/pih/warehouse/api/LocationApiController.groovy

```diff
@@ -271,7 +271,7 @@ class LocationApiController extends BaseDomainApiController {
             throw new IllegalArgumentException("File cannot be empty")
         }
 
-        if (importFile.fileItem.contentType != "text/csv") {
+        if (importFile.contentType != "text/csv") {
```

Comment:
Is this a verified API change for uploaded files? If so, let's include a link to that documentation in the PR and/or ticket.

Reply:
We changed it many times, there is a ticket for investigating the rest of those places with `fileItem`: [ticket](https://pihemr.atlassian.net/browse/OBGM-669?atlOrigin=eyJpIjoiMWY2MWNiMzgwMjFjNDMzMzg3ZjNjNTY0ZGQzYjgwMDgiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)

ImportFile is a type of MultipartFile - on this instance, we can call getContentType, getSize, getBytes, etc.
Docs: https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/multipart/MultipartFile.html